dmidecode ONIE cross compile patch

Copyright (C) 2017 Curt Brune <curt@cumulusnetworks.com>

SPDX-License-Identifier:     GPL-2.0

Use $(CROSS_COMPILE)gcc as the compiler so dmidecode cross-compiles in
the ONIE build environment.  (dmidecode's Makefile honors CFLAGS/LDFLAGS
from the environment, so no further changes are needed.)
diff --git a/Makefile b/Makefile
index 985992e..ae70e4d 100644
--- a/Makefile
+++ b/Makefile
@@ -12,7 +12,7 @@
 #   (at your option) any later version.
 #
 
-CC     ?= gcc
+CC     = $(CROSS_COMPILE)gcc
 # Base CFLAGS can be overridden by environment
 CFLAGS ?= -O2
 # When debugging, disable -O2 and enable -g
